combat-trainer.lic

```diff
@@ -197,9 +197,18 @@ class LootProcess
 
     @loot_specials = settings.loot_specials
     echo("  @loot_specials: #{@loot_specials}") if $debug_mode_ct
+    @dump_junk = settings.dump_junk
```

Comment:
Please add an echo that outputs the state of this setting (see line 199) for an example.